library/src/main/android/permissions/dispatcher/PermissionUtils.java

```diff
@@ -94,13 +96,23 @@ public static boolean hasSelfPermissions(Context context, String... permissions)
      * @see #hasSelfPermissions(Context, String...)
      */
     private static boolean hasSelfPermission(Context context, String permission) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.M && Build.MANUFACTURER.equalsIgnoreCase("Xiaomi")) {
```

Comment:
The internet seems to agree that this is sufficient. I guess there will be issues opened in case there actually _is_ such a device. I'm fine with this for now!